docs/conf.py

```diff
@@ -58,7 +59,7 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ['.rst', ]
```

Comment:
Do we want to add .md here? m2r's docs say it should read `['.rst', '.md']`, but sphinx says it should be 
```
source_suffix = {
    '.rst': 'restructuredtext',
    '.md': 'markdown',
}
```

The latter seems to be a more robust syntax, though the docs don't appear to look any different with any of the three options.